src/Nancy.Bootstrappers.Unity/UnityNancyBootstrapper.cs

```diff
@@ -81,7 +81,11 @@ protected override INancyEngine GetEngineInternal()
         /// <returns>Container instance</returns>
         protected override IUnityContainer GetApplicationContainer()
         {
-            return new UnityContainer();
+            var container = new UnityContainer();
+
+            container.AddNewExtension<EnumerableExtension>();

```

Comment:
My app injects the `IUnityContainer` because it has to reuse an existing container from the application at large.

```c#
public class DiBootstrapper : UnityNancyBootstrapper
{
    private readonly IUnityContainer _container;

    public DiBootstrapper(IUnityContainer container)
    {
        _container = container;
    }

    protected override IUnityContainer GetApplicationContainer()
    {
        return _container;
    }
}
```

Would I have to then manually add the extension or could we make this more extendable?